test_format/format.eno

```diff
@@ -2336,3 +2336,135 @@ com.example.my-app)
   :ccc
   :ddd}
 --Expected
+
+# GitHub Issue #193 - rule 3 alignment for unquote
+
+--Input
+(foo ~'@(bar)
+    baz)
+
+(foo ~'@(bar)
+     baz)
+
+(foo ~'@(bar)
+      baz)
+
+(foo ~'@(bar)
+       baz)
+
+(foo ~'@(bar)
+        baz)
+
+(foo ~'@(bar
+         baz))
+--Input
+
+--Expected
+(foo ~'@(bar)
+  baz)
+
+(foo ~'@(bar)
+     baz)
+
+(foo ~'@(bar)
+  baz)
+
+(foo ~'@(bar)
+  baz)
+
+(foo ~'@(bar)
+  baz)
+
+(foo ~'@(bar
+          baz))
+--Expected
+
+# GitHub Issue #193 - rule 3 alignment for metadata
+
+--Input
+(defonce ^:private
+         foo)
+
+(defonce ^:private
+          foo)
+
+(defonce ^ :private
+         foo)
+
+(defonce ^ :private
+           foo)
+
+(defonce ^ :private
+          foo)
+
+(defonce ^{:private
+          foo})
+
+(defonce ^{:private
+           foo})
+
+(defonce ^{:private
+            foo})
+--Input
+
+--Expected
+(defonce ^:private
+         foo)
+
+(defonce ^:private
+          foo)
+
+(defonce ^ :private
+         foo)
+
+(defonce ^ :private
+           foo)
+
+(defonce ^ :private
+  foo)
+
+(defonce ^{:private
+           foo})
+
+(defonce ^{:private
+           foo})
+
+(defonce ^{:private
+           foo})
+--Expected
+
+# GitHub Issue #193 - rule 3 alignment for ignore form
```

Comment:
Perhaps also add cases where the ignore macro is not _directly_ before bar:

```clojure
#_ bar

#_
bar


     #_
bar

#_
   bar
```

Reply:
Minor nit: I believe the official reference calls it the 'discard' macro, not 'ignore'

Reply:
Thank you both for the suggestions :+1: